lib/plausible/verification/diagnostics.ex

```
@@ -150,13 +151,77 @@ defmodule Plausible.Verification.Diagnostics do
+         "https://plausible.io/docs/troubleshoot-integration"}
+      ]
+    }
+  end
+
+  def interpret(
+        %__MODULE__{
+          snippets_found_in_body: 0,
+          snippets_found_in_head: 1,
+          plausible_installed?: true,
+          wordpress_likely?: true,
+          proxy_likely?: false,
+          wordpress_plugin?: false,
+          callback_status: -1
+        },
+        _url
+      ) do
+    %Result{
+      ok?: false,
+      errors: ["We encountered a problem trying to verify your website"],
+      recommendations: [
+        {"The integration may be working but as you're running an older version of our script, we cannot verify it automatically. Please install our WordPress plugin to use the built-in proxy",
+         "https://plausible.io/wordpress-analytics-plugin"}
+      ]
+    }
+  end
+
+  def interpret(
+        %__MODULE__{
+          snippets_found_in_body: 0,
+          snippets_found_in_head: 1,
+          plausible_installed?: true,
+          wordpress_likely?: true,
+          proxy_likely?: false,
+          wordpress_plugin?: true,
+          callback_status: -1
+        },
+        _url
+      ) do
+    %Result{
+      ok?: false,
+      errors: ["We encountered a problem trying to verify your website"],
+      recommendations: [
+        {"The integration may be working but as you're running an older version of our script, we cannot verify it automatically. Please disable and then enable the proxy in our WordPress plugin, then clear your WordPress cache",
+         "https://plausible.io/wordpress-analytics-plugin"}
+      ]
+    }
+  end
+
+  def interpret(
+        %__MODULE__{
+          plausible_installed?: true,
+          callback_status: 0,
           proxy_likely?: true
         },
         _url
-      )
-      when callback_status != 202 do
```

So the case of callback failing due to other HTTP code will be handled by the final, catch-all clause, right? (like 500/503s, however unlikely).

---

Yes, I'll add a test.

---

d88366ca7